### DISCUSSION TITLE
Is it possible to list the tracks of an album during import?

### DISCUSSION THREAD

**Discussion by guerda** [beetbox/beets#5267] [Q&A] [answered]
During import of albums, it would be helpful to have a peek at the tracks which are in the album to be imported. Right now, you first have to select an album candidate, before you can see the tracks beets wants to import.
It would be helpful to have an option to print all tracks first and then select a candidate.

For me this would be very helpful to decide better on the album candidates.
Is this possible to configure? or is this a feature request?

**Comment by DjSlash** [comment, marked as answer]
When beets has a few possibilities as potential matches, it gives you also a few options to choose from:
```
➜ # selection (default 1), Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort, Print tracks, Open files with Picard?
```
(This output is from version 2.0.)

Here you can choose `P` for "Print tracks", and it will show you the tracks.

**Reply by guerda** [reply]
Oh great! It looks like I'm running an outdated version of beets. Thank you!
